tests/transforms/test_convert_rois.py

```diff
@@ -80,7 +62,7 @@ def test_binarize_and_convert_rois_schema(s2p_stat_fixture,
                               'y': 0,
                               'height': 2,
                               'width': 3,
-                              'valid_roi': False,
+                              'valid_roi': True,
```

Comment:
Quick explanation for why this test case got changed?

Reply:
I believe the original code for motion border was incorrect.

Reply:
I see. So the test case was incorrect.